integreat_cms/cms/forms/pages/page_filter_form.py

```diff
@@ -53,6 +78,12 @@ def apply(self, pages, language_slug):
                 pages = self.filter_by_query(pages, language_slug)
             if "translation_status" in self.changed_data:
                 pages = self.filter_by_translation_status(pages, language_slug)
+            if "status" in self.changed_data:
+                pages = self.filter_by_latest_revision(pages, language_slug)
+            if "date_from" in self.changed_data:
+                pages = self.filter_by_start_date(pages, language_slug)
+            if "date_end" in self.changed_data:
```

Comment:
```suggestion
            if "date_to" in self.changed_data:
```

I think this was a typo...